pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1144,13 +1145,15 @@ class Build {
                 }
 
                 // Sign and archive jobs if needed
-                try {
-                    context.timeout(time: buildTimeouts.SIGN_JOB_TIMEOUT, unit: "HOURS") {
-                        sign(versionInfo)
+                if (enableSigner) {
+                    try {
+                        context.timeout(time: buildTimeouts.SIGN_JOB_TIMEOUT, unit: "HOURS") {
+                            sign(versionInfo)
+                        }
+                    } catch (FlowInterruptedException e) {
+                        context.println "[ERROR] Sign job timeout (${buildTimeouts.SIGN_JOB_TIMEOUT} HOURS) has been reached. Exiting..."
+                        throw new Exception()
                     }
-                } catch (FlowInterruptedException e) {
-                    context.println "[ERROR] Sign job timeout (${buildTimeouts.SIGN_JOB_TIMEOUT} HOURS) has been reached OR the downstream sign job failed. Exiting..."
```

Comment:
In your rebase, did you pull in the latest commits as I don't think this relates to the commit's you're trying to revert?

Reply:
yes. this is not relevent. Corrected.